sdk/storage/azure_storage_blob_test/src/lib.rs

```diff
@@ -1,10 +1,22 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
-use azure_core::http::ClientOptions;
+use azure_core::{
+    http::{ClientOptions, RequestContent, Response},
+    Result,
+};
 use azure_core_test::Recording;
+use azure_storage_blob::{
+    models::BlockBlobClientUploadResult, BlobClient, BlobClientOptions, BlobContainerClient,
+    BlobContainerClientOptions, BlobServiceClient, BlobServiceClientOptions,
+};
 
-pub fn recorded_test_setup(recording: &Recording) -> (ClientOptions, String) {
+/// Takes in a Recording instance and returns an instrumented options bag and endpoint.
+///
+/// # Arguments
+///
+/// * `recording` - A reference to a Recording instance.
+pub fn _recorded_test_setup(recording: &Recording) -> (ClientOptions, String) {
```

Comment:
Why prefaced with an underscore? If internal, just remove `pub` or make it `pub(crate)`?